src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -61,6 +61,24 @@ public ResponseEntity<Object> getAllIssuers(@RequestParam(required = false) Stri
         return ResponseEntity.status(HttpStatus.OK).body(responseWrapper);
     }
 
+    @GetMapping("/{issuer-id}/wellknown")
+    public ResponseEntity<Object> getIssuerWellknown(@PathVariable("issuer-id") String issuerId) throws ApiNotAccessibleException, IOException {
+        ResponseWrapper<Object> responseWrapper = new ResponseWrapper<>();
+        IssuerDTO issuerConfig;
+        RestTemplate restTemplate = new RestTemplate();
+        try {
+            issuerConfig = issuersService.getIssuerConfig(issuerId);
+            String wellknown = issuerConfig.wellKnownEndpoint;
+
+            ResponseEntity<String> responseEntity = restTemplate.getForEntity(wellknown, String.class);
+            return ResponseEntity.status(responseEntity.getStatusCode()).body(responseEntity.getBody());
```

Comment:
Status code we can directly use HttpStatus.OK

Reply:
Return type should be response wrapper. In case if success response should be the field, in case of error response is empty.
Please refer issuer config api for this

Reply:
Fetching wellknowm response should be delegated to IssuerService